README.md

```diff
@@ -2,23 +2,18 @@
 
 ### Instructions
 
-1. Complete the tasks below ahead of a technical interview, where we'll ask you to explain and expand on your your solution.
-1. Provide your solution in JavaScript or TypeScript.
-1. Spend no more than 2 hours on the exercise.
-1. Write your solution as if this code was going into production - we want to see how you'd deliver something in a real world situation. How can you be confident this code will work as expected? What will happen in the case of unexpected input?
-1. Read the _Background_ section below carefully for any performance implications. Consider how to optimise your solution for best performance.
-1. Make use of official documentation and online resources. Feel free to use any resources you want (for example, Stack Overflow), but please make a note of them in your solution.
-1. Provide the solution as a link to a git repository, make sure to commit regularly to allow us to understand your thought processes and decision making as you build up the program.
+Below we have outline a simple Node JS technical test that you should approach as a pairing exercise with the interviewer. What we're looking for is good discussion on the approaches you take to complete the task. There is no pressure to write incredible code or even finish the task, it's all about approach.
```

Comment:
Maybe remove the world `simple`? Don't want people to feel bad if they get stuck